tools/archetypes/default.html

```diff
@@ -1,9 +1,5 @@
 <!--*- coding: utf-8 -*- html -*- directory_name/file_name.html ****************
```

Comment:
```suggestion
<!--*- coding: utf-8 -*- html -*- directory_name/file_name.html ****************
```

this was written by me a really, really long time ago and i now forget exactly what it is, but it seems most likely something to do w/ emacs:

🔗 https://www.gnu.org/software/emacs/manual/html_node/emacs/Choosing-Modes.html